testcase/ninja_symlink_outputs.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Copyright 2020 Google Inc. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+mk="$@"
+
+cat <<EOF >Makefile
+dangling_symlink:
+	ln -sf nil dangling_symlink
```

Comment:
Does it make sense to differentiate these two? After all, Kati doesn't run the build so it can't be affected by whether a symlink is dangling or not.

Reply:
Yeah, the resolved symlink statement has an input file. The dangling symlink doesn't.